html/cross-origin-embedder-policy/header-parsing.https.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <meta charset="utf-8">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+'use strict';
+function createIframe(t, values) {
+  const parent = document.createElement('iframe');
+  const child = document.createElement('iframe');
+  const params = new URLSearchParams(values.map((value) => ['value', value]));
+  parent.setAttribute('src', `resources/empty-coep.py?${params}`);
+  document.body.appendChild(parent);
+  t.add_cleanup(() => parent.remove());
+
+  return new Promise((resolve, reject) => {
+      parent.onload = resolve;
+      parent.onerror = reject;
```

Comment:
reject()ing with an Error (possibly containing the URL or src attribute) would make any failures here more comprehensible.

Reply:
Sounds good